src/valkey-check-rdb.c

```diff
@@ -280,7 +674,13 @@ int redis_check_rdb(char *rdbfilename, FILE *fp) {
                 decrRefCount(auxkey);
                 goto eoferr;
             }
-
+            if (!strcasecmp(auxkey->ptr, "lua")) {
+                /* In older version before 7.0, we may save lua scripts in a replication RDB,
+                 * although it is not an actually aux field, we will still print it in here since
+                 * it's easy to filter using external grep. Use a counter so that at the end we
+                 * can print its number, if any. */
```

Comment:
Some parts of this comment is from #1448, not very relevant here?

```suggestion
                /* In older version before 7.0, we may save lua scripts in a replication RDB. */
```